src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -1295,7 +1302,9 @@ static void
 pretty_print_test_desc(const boost::property_tree::ptree& test, int& test_idx,
                        std::ostream & _ostream, const std::string& bdf)
 {
-  if (test.get<std::string>("status", "").compare(test_token_skipped)) {
+  // If the status is anything other than skipped print the test name
+  auto _status = test.get<std::string>("status", "");
+  if (!boost::iequals(_status, test_token_skipped)) {
```

Comment:
Hmm...why the case insensitive comparison.   By doing so, you have indirectly added case insensitivity support, something to which the code currently doesn't support.

Reply:
Makes sense Ill change it to boost::equals, I thought iequals meant is equal haha. I believe in this section of the code we are case insensitive to begin with. I remember you brought it up before, maybe we should make a CR or talk about it?